content/en/news/releases/1.11.x/announcing-1.11/change-notes/index.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Change Notes
+linktitle: 1.11.0
+subtitle: Minor Release
+description: Istio 1.11.0 release notes.
+publishdate: 2020-08-10
+release: 1.11.0
+weight: 10
+aliases:
+    - /news/announcing-1.11.0
+---
+
+{{< warning >}}
+This is an automatically generated rough draft of the release notes and has not yet been reviewed.
+{{< /warning >}}
+
+# Release Notes
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/about/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+
+
+## Traffic Management
+
+
+- **Promoted** [CNI](/docs/setup/additional-setup/cni/) to beta ([Issue #86](https://github.com/istio/enhancements/issues/86)).
```

Comment:
Spacing is inconsistent between bullets. Added suggestions to make it consistent.

Reply:
Also trailing `.`